tests/security/atsec/dbus_fuzzer.pm

```diff
@@ -0,0 +1,113 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run 'DBus fuzzer' test case of ATSec test suite
+# Maintainer: xiaojing.liu <xiaojing.liu@suse.com>
+# Tags: poo#109978
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use atsec_test;
+use Mojo::Util 'trim';
+use Data::Dumper;
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    # Install the required packages
+    zypper_call('in glib2-devel libffi-devel gcc make');
+
+    # Compile
+    assert_script_run("cd $atsec_test::code_dir/pentest/dfuzzer-master/src");
+    assert_script_run('make');
```

Comment:
`dfuzzer` kind of switched to `meson` in https://github.com/matusmarhefka/dfuzzer/pull/26 and can be built and run with
```
meson --buildtype=release build
ninja -C ./build
./build/dfuzzer ...
```
`Makefile` will be removed once the `dfuzzer` package switches to `meson` as well. Also it no longer depends on `libffi-devel`.